build.php

```diff
@@ -26,32 +26,45 @@
 use Seld\PharUtils\Timestamps;
 use Symfony\Component\Finder\Finder;
 
-require_once __DIR__.'/vendor/autoload.php';
+$dir = __DIR__;
+$build_dir = $dir.'/build';
+$outpath = $build_dir.'/kint.phar';
 
-\mkdir(__DIR__.'/build');
+require_once $dir.'/vendor/autoload.php';
 
-$outpath = __DIR__.'/build/kint.phar';
+if (ini_get('phar.readonly') == 1) {
+    die("Error: Creating phar files is disabled by your system.\nYou must set \"phar.readonly = Off\" in your php.ini to continue.\n");
```

Comment:
I think the PHP builtin error message is clear enough, and will get more search hits if people google it:

> PHP Fatal error:  Uncaught UnexpectedValueException: creating archive ".../kint/build/kint.phar" disabled by the php.ini setting phar.readonly in .../kint/build.php:36